golden_response/staging/get_triples/place_type.json

```diff
@@ -495,7 +2444,7 @@
       "subjectId": "country/TCA",
       "subjectName": "Turks and Caicos Islands",
       "subjectTypes": [
-        "AdministrativeArea",
+        "AdministrativeArea1",
```

Comment:
is this expected?

Reply:
Yes because it is a british overseas territory as well. So AA1 is probably more accurate than AA.